pkg/agent/util/net.go

```diff
@@ -136,3 +140,21 @@ func ContainIPv6Addr(ips []net.IP) bool {
 	}
 	return false
 }
+
+func GetIPWithFamily(ips []net.IP, addrFamily uint8) (net.IP, error) {
+	if addrFamily == FamilyIPv6 {
+		for _, ip := range ips {
+			if ip.To4() == nil {
+				return ip, nil
+			}
+		}
+		return nil, errors.New("no IP found with the specified AddressFamily")
```

Comment:
```suggestion
		return nil, errors.New("no IP found with IPv6 AddressFamily")
```
Shall we tell which AddressFamily it is to make the error info more detailed?